api-1.11/src/main/java/org/openmrs/module/fhir/helper/AllergyHelperImpl1_11.java

```diff
@@ -0,0 +1,104 @@
+package org.openmrs.module.fhir.helper;
+
+import ca.uhn.fhir.rest.server.exceptions.MethodNotAllowedException;
+import ca.uhn.fhir.rest.server.exceptions.ResourceNotFoundException;
+import org.hl7.fhir.dstu3.model.AllergyIntolerance;
+import org.openmrs.Patient;
+import org.openmrs.activelist.Allergy;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.api.APIException;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.helper.AllergyHelper;
+import org.openmrs.module.fhir.api.util.FHIRConstants;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+import org.openmrs.module.fhir.util.FHIRAllergyIntoleranceUtil1_11;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@Component(value = "fhir.AllergyHelper")
+@OpenmrsProfile(openmrsVersion = "1.11.*")
+public class AllergyHelperImpl1_11 implements AllergyHelper {
+
+	@Autowired
+	private PatientService patientService;
```

Comment:
you should use one convention - autowired or use context

Reply:
I decide to use the OpenMRS Context to avoid issues with beans injection.

Reply:
ok, remove unused variable